src/app/store/actions/app.actions.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction, props } from '@ngrx/store';
+
+export const TestApp = createAction(
+  '[App] Test App'
```

Comment:
move string to constant

Reply:
In the future I will, but it is just a test action.